RP2040Kit.ato

```diff
@@ -4,66 +4,52 @@ import Resistor from "generics/resistors.ato"
 import Capacitor from "generics/capacitors.ato"
 import ButtonPullup from "generics/buttons.ato"
 
-import I2C from "generics/interfaces.ato"
 import Power from "generics/interfaces.ato"
+import I2C from "generics/interfaces.ato"
 import QSPI from "generics/interfaces.ato"
 import SPI from "generics/interfaces.ato"
 import USB2 from "generics/interfaces.ato"
 import UART from "generics/interfaces.ato"
 
 
-module RP2040Kit:
+module RP2040_3v3_Power:
     # Microcontroller
     micro = new RP2040
 
-    power = new Power
-    power.vcc ~ micro.vcc
-    power.gnd ~ micro.gnd
+    # This kit uses 3.3v for power
+    power_3v3 = new Power
 
-    power_1V = new Power
-    power_1V.vcc ~ micro.dvdd
-    power_1V.gnd ~ micro.gnd
+    # Apply power to the RP2040
+    power_3v3 ~ micro.usb_phy_power
+    power_3v3 ~ micro.iovdd_power
 
-    i2c = new I2C
-    i2c.sda ~ micro.gpio20
-    i2c.scl ~ micro.gpio21
+    # Bypass capacitors 100nF - 3V3
+    bypass_cap_100nF_1 = new BypassCap100nF
+    bypass_cap_100nF_2 = new BypassCap100nF
+    bypass_cap_100nF_3 = new BypassCap100nF
+    bypass_cap_100nF_4 = new BypassCap100nF
+    bypass_cap_100nF_5 = new BypassCap100nF
+
+    power_3v3 ~ bypass_cap_100nF_1.power
+    power_3v3 ~ bypass_cap_100nF_2.power
+    power_3v3 ~ bypass_cap_100nF_3.power
+    power_3v3 ~ bypass_cap_100nF_4.power
+    power_3v3 ~ bypass_cap_100nF_5.power
 
+    # Bypass capacitors 1uF - 3V3
+    bypass_cap_1uF = new BypassCap1uF
+    power_3v3 ~ bypass_cap_1uF.power
+
+module USB_Powered_RP2040Kit from RP2040_3v3_Power:
     osc = new _Oscillator
     osc.input ~ micro.xin
     osc.output ~ micro.xout
     osc.gnd ~ micro.gnd
 
-    # UART interface
```

Comment:
We might want to mimic these interfaces a layer up in order to keep the projects currently using the rp2040 happy and make future references to them terse?

Reply:
I agree - I just put that there as a test to see if I could add it.  Wasn't ment to be in the final PR.